### PR TITLE
ps2:week 11: transfer functionality added

### PR DESCRIPTION
**Pull request by Pulkit007**
## Issue that this pull request solves

Closes: # (issue number)

## Types of changes

_Put an `x` in the boxes that apply_

-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [ x] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] Other (please describe):

## Checklist

_Put an `x` in the boxes that apply_

-   [ ] My code follows the [style guidelines of this project](https://docs.google.com/document/d/1GI2Hile8UbGZ82gFe1y4wAVPcNPXip1cmXTzog1S41U/edit)
-   [ x] I have performed a self-review of my own code
-   [ ] I have created new branch for this pull request
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation
-   [ ] My changes generate no new warnings
-   [ ] My changes does not break the current system and it passes all the current test cases.

## Screenshots

Please attach the screenshots of the changes made in case of change in user interface

## Other information

Any other information that is important to this pull request

### CODE REVIEW

**Comment by Avi2002Soy** [comment]
@PranshuNayak @samay-rgb kindly review the changes @akshatnema merge the PR

**Comment by rakshit512002** [comment]
LGTM @akshatnema

**Review by samay-rgb — APPROVED** [review]

File: FusionIIIT/applications/ps2/models.py [hunk 1]
```diff
@@ -21,4 +21,16 @@ class StockEntry(models.Model):
...
+        db_table = 'ps2_stocks'
+
+class TransferEntry(models.Model):
+    Item_id = models.IntegerField(blank=False)
+    From_department = models.CharField(max_length=50, blank=False)
+    From_location = models.IntegerField(blank=True, null=True)
+    To_department = models.CharField(max_length=50, blank=False)
+    To_location = models.IntegerField(blank=True, null=True)
+    Date = models.DateTimeField(blank=False)
+    Remark = models.CharField(max_length=200, blank=False)
```
Comment (PranshuNayak) [inline FusionIIIT/applications/ps2/models.py:33]: @Pulkit007 field names should be lowercase.

File: FusionIIIT/applications/ps2/models.py [hunk 2]
```diff
@@ -21,4 +21,16 @@ class StockEntry(models.Model):
...
 
     class Meta:
-        db_table = 'ps2_stocks'
\ No newline at end of file
+        db_table = 'ps2_stocks'
+
+class TransferEntry(models.Model):
+    Item_id = models.IntegerField(blank=False)
+    From_department = models.CharField(max_length=50, blank=False)
+    From_location = models.IntegerField(blank=True, null=True)
```
Comment (PranshuNayak) [inline FusionIIIT/applications/ps2/models.py:29]: Why is this an Integerfield?

**Review by PranshuNayak — COMMENTED** [review]
Register the model on admin.py so that it can be viewed on django admin if needed.

File: FusionIIIT/applications/ps2/models.py (hunk 2 above)
Comment (Pulkit007) [reply FusionIIIT/applications/ps2/models.py:29]: Currently ID is auto generated from Django itself and it is integer

**Review by PranshuNayak — APPROVED** [review]
LGTM! 
@akshatnema db changes are fine.

**Review by samay-rgb — APPROVED** [review]

File: FusionIIIT/templates/ps2/viewtransfers.html [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+            <select name="d">
+                {% for i in department %}
+                <option value="{{i}}">{{i}}</option>
+                {% endfor %}
+            </select>
+            <input class="ui large primary button right floated" type="submit" name="submit" onclick="">
+                Submit
+            </input>
+        </div>
+    </form>  -->
```
Comment (akshatnema) [inline FusionIIIT/templates/ps2/viewtransfers.html]:
```suggestion
```

**Pull request merged**